pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -234,7 +236,11 @@ def test_build_series(self):
                 ),
             ]
         )
-        assert result == expected
+
+        if PY35:
```

Comment:
can you encapsulate this in a function, assert_results_equal(result, expected) (do it in this file i think)

Reply:
can you do this here

Reply:
Sorry missed this before but should be done now